### PR TITLE
created new feature for featuring products, orders, and category

### PR DESCRIPTION
**Pull request by lucpizz**
## Proposed changes

The isFeatured attribute will allow admins, merchants to feature brands, categories, and products

## PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/davidsaulrodriguez/shop-portal/blob/master/CONTRIBUTING.md) doc
- [x] I have read the [BRANCHING MODEL](https://github.com/davidsaulrodriguez/shop-portal/blob/master/BRANCHINGMODEL.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## PR Type

What types of changes does your code introduce to **Shop Portal**?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code Style Update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (Please describe):

## Current behavior

Issue Number: #52 

## New behavior

Fixes: #52 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Further comments

N/A

### CODE REVIEW

**Review by Delph-Sunny — COMMENTED** [review]
CategoryModel should have isFeatured default set to false

File: models/categoryModel.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ const CategorySchema = new Schema({
     type: Boolean,
     default: true,
   },
+  isFeatured: {
+    type: Boolean,
+    default: true,
```
Comment (davidsaulrodriguez) [inline models/categoryModel.js]: This does need to be false. Not every category will be featured. That will be at the discretion of the admin.

**Review by Delph-Sunny — DISMISSED** [review]
Tutto bene!

File: models/categoryModel.js (hunk 1 above)
Comment (lucpizz) [reply models/categoryModel.js]: changed isFeatured to false

**Comment by lucpizz** [comment]
I removed the seed scripts that caused the 3 security concerns.  They are no longer needed.  Tests passed.

**Review by davidsaulrodriguez — DISMISSED** [review]
Let's Get It Merged

**Review by davidsaulrodriguez — APPROVED** [review]
All approved.

**Review by Delph-Sunny — APPROVED** [review]
Ok

**Pull request merged**